manifests/install.pp

```diff
@@ -71,9 +71,17 @@
 
   # if the user didn't specify a version, just use "installed".
   # if they did specify a version, assume they provided a valid one
-  $_version = $version ? {
-    'LATEST' => 'installed',
-    default  => $version,
+  case $::osfamily {
+    'Debian': {
+      $_version = 'installed'
```

Comment:
Am I reading this correctly that Debian and Ubuntu users will be unable to specify versions to install?